libraries/AP_HAL_ChibiOS/hwdef/iomcu-f103-dshot/hwdef.inc

```diff
@@ -0,0 +1,93 @@
+# hw definition file for processing by chibios_pins.py
+
+include ../iomcu/hwdef.inc
```

Comment:
This needs to almost exactly match the old hwdef.dat. You have added a whole bunch of extra lines - please remove them

Reply:
removed them

Reply:
> This needs to almost exactly match the old hwdef.dat. You have added a whole bunch of extra lines - please remove them
remove them